doc/community.rst

```diff
@@ -3,7 +3,7 @@
 NEST Community
 ===============
```

Comment:
Is the underlining one character too long?

Reply:
Yes, fixed it.